build/operations/services/create-appi.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
should we rename the file to "create-app-insights" for more clarity when browsing the file structure?

Reply:
The names would get crazy long for different services.  So each file uses the recommended short abbr, except for a couple services that don't have one.  This is one reason I clearly add the name and description of the service in the top of each file.  Additionally, these shouldn't be called directly only through a recipe.